packages/documentation/copy/en/handbook-v2/Classes.md

```diff
@@ -250,7 +250,7 @@ TypeScript has some special inference rules for accessors:
 
 - If `get` exists but no `set`, the property is automatically `readonly`
 - If the type of the setter parameter is not specified, it is inferred from the return type of the getter
-- Getters and setters must have the same [Member Visibility](#member-visibility)
+- `set` accessors may be less visible than their corresponding `get` accessor

```

Comment:
I think this line could just be deleted if it's no longer true.

Reply:
(the old one, I mean)

Reply:
Thanks! I've removed this line in the latest commit.